### PR TITLE
US133014 - switch to new multiple choice API

### PR DESCRIPTION
**Pull request by Scy-D2L**
Update the multiple-choice version of the component to accept the updated API.
Includes new demo data and updated test cases.

### CODE REVIEW

File: components/d2l-questions-multiple-choice.js [hunk 1]
```diff
@@ -9,16 +9,19 @@ import { getUniqueId } from '@brightspace-ui/core/helpers/uniqueId.js';
 import { LocalizeDynamicMixin } from '@brightspace-ui/core/mixins/localize-dynamic-mixin.js';
 import { radioStyles } from '@brightspace-ui/core/components/inputs/input-radio-styles.js';
 
-//Database is using magic numbers
-const IS_CHECKED = '1';
-const IS_NOT_CHECKED = '0';
+const REL_IDENTIFIER = 'https://questions.api.brightspace.com/rels/identifier';
```
Comment (vix633) [inline components/d2l-questions-multiple-choice.js:12]: Do we want these to live in the consts file?

File: components/d2l-questions-multiple-choice.js [hunk 2]
```diff
@@ -75,74 +78,121 @@ class D2lQuestionsMultipleChoice extends LocalizeDynamicMixin(LitElement) {
...
-			const answer = answers.find(answer => answer.properties && answer.properties.atomId === choice.properties.atomId);
-			return { ...choice.properties, ...answer.properties };
-		});
+	async updated(changedProperties) {
+		super.updated();
+		if ((changedProperties.has('question') || changedProperties.has('questionResponse'))) {
+			try {
+				this._loadChoices();
+			} catch (e) {
+				console.error('d2l-questions-multiple-choice: Unable to load choices from question');
```
Comment (vix633) [inline components/d2l-questions-multiple-choice.js:99]: We have `e` here, do we wanna do anything with it? Also, if we don't re-throw here, our unhandled exception handler wont be able to grab this and log it.

**Review by JamieTran — APPROVED** [review]
lgtm

File: components/d2l-questions-multiple-choice.js (hunk 2 above)
Comment (Scy-D2L) [reply components/d2l-questions-multiple-choice.js]: Updated: now we're still logging the error in the browser and also throwing it for the handler. :)

File: components/d2l-questions-multiple-choice.js (hunk 1 above)
Comment (Scy-D2L) [reply components/d2l-questions-multiple-choice.js:12]: Will do this in another PR

**Review by amauryperez — COMMENTED** [review]
LGTM

**Pull request merged**